src/main/resources/requests/hearingRequests.ts

```diff
@@ -18,5 +18,21 @@ export class HearingRequests {
     }
     return [];
   }
+
+  public async getSubscriptionCaseDetails(caseReferenceNo: string): Promise<Array<any>> {
```

Comment:
1. you are not getting subscription, you are getting hearing by case reference number
2. promise should return array of hearings (we have a model for that) instead of any

Reply:
Change the function name.